triage/update_summaries.sh

```diff
@@ -72,7 +72,9 @@ mkdir -p slices
   --builds triage_builds.json \
   --tests triage_tests/*.json \
   --output failure_data.json \
-  --output_slices slices/failure_data_PREFIX.json
+  --output_slices slices/failure_data_PREFIX.json \
+  --logtostderr=true \
```

Comment:
I'm pretty sure this is wrong but I'm not sure how else to pass in the flag

Reply:
binaries frequently set this themselves / hide the flag

Reply:
https://cs.k8s.io/?q=logtostderr%22&i=nope&files=&repos=

Reply:
Setting it internally does not make the flag work

Reply:
I switched to klog/v2 _and_ called InitFlags, that should do it

Reply:
Added a custom flagset as per https://github.com/kubernetes/klog/blob/master/examples/coexist_glog/coexist_glog.go

Reply:
Ok, it seems to have passed without any issues